services/brig/src/Brig/User/Search/Index/Types.hs

```diff
@@ -37,49 +40,57 @@ instance Exception IndexError
 
 newtype IndexVersion = IndexVersion {docVersion :: DocVersion}
 
--- | Represents the searchable subset of a user record, as indexed by ES
+-- | Represents an ES *document*, ie. the subset of user attributes stored in ES.
+-- See also 'IndexUser'.
 --
--- If a user is not searchable, e.g. because she opted out or the account got
+-- If a user is not searchable, e.g. because the account got
 -- suspended, all fields except for the user id are set to 'Nothing' and
 -- consequently removed from the index.
 data UserDoc
   = UserDoc
       { udId :: UserId,
-        udName :: Maybe Text,
+        udTeamId :: Maybe TeamId,
+        udName :: Maybe Name,
         udNormalized :: Maybe Text,
-        udHandle :: Maybe Text,
+        udHandle :: Maybe Handle,
         udColourId :: Maybe ColourId
       }
+  deriving (Eq, Show)
 
 instance ToJSON UserDoc where
   toJSON ud =
-    object $
-      "id" .= udId ud
-        # "name" .= udName ud
-        # "normalized" .= udNormalized ud
-        # "handle" .= udHandle ud
-        # "accent_id" .= udColourId ud
-        # []
+    object
+      [ "id" .= udId ud,
+        "team_id" .= udTeamId ud,
```

Comment:
Should we also standardise on `team` in ES?

Reply:
Yeah, let's keep it consistent.